src/components/ToastCard/_toast-card.scss

```diff
@@ -1,4 +1,10 @@
 @import "vanilla-framework/scss/vanilla";
+@import "../../scss/functions/z-index";
+
+.toast-container {
+  // These cards need to appear above modals and panels.
+  z-index: z("zelda") + 3 !important;
+}
```

Comment:
The cards were being show below panels, such as on the access panel when adding a user.